go/consensus/tendermint/abci/mux.go

```diff
@@ -288,23 +311,7 @@ func (mux *abciMux) Info(req types.RequestInfo) types.ResponseInfo {
 }
 
 func (mux *abciMux) SetOption(req types.RequestSetOption) types.ResponseSetOption {
```

Comment:
Is this still needed? (if we skipped it, the embedded impl. from `types.BaseApplication` would be used right?)